packages/js/product-editor/src/utils/register-product-editor-block-type.ts

```diff
@@ -15,12 +14,17 @@ interface BlockRepresentation< T extends Record< string, object > > {
 	settings: Partial< BlockConfiguration< T > >;
 }
 
+// Define a more generic type for the select function to avoid TypeScript errors
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type SelectType = ( store: string ) => Record< string, unknown >;
+
 export function useEvaluationContext( context: Record< string, unknown > ) {
 	const { postType } = context;
 
 	const productId = useEntityId( 'postType', postType );
 
-	const getEvaluationContext = ( select: typeof WPSelect ) => {
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
With the `any` type removed, we no longer need this comment.

Reply:
Right. Removed in 6b5aa8b3a5.